README.md

```diff
@@ -32,29 +32,49 @@ Mycroft is a hackable open source voice assistant.
 
 ## Getting Started
 
-First, get the code on your system!  The simplest method is via git ([git installation instructions](https://gist.github.com/derhuerst/1b15ff4652a867391f03)):
-- `cd ~/`
-- `git clone https://github.com/MycroftAI/mycroft-core.git`
-- `cd mycroft-core`
-- `bash dev_setup.sh`
+Mycroft might be packaged by your distribution already, in that case installing it might be as simple as `apk add mycroft-core` or equivalent for your distribution.
```

Comment:
I'm happy for us to add a community releases section, but it can't be the primary recommendation. We're also probably better off pointing to the [main docs](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/get-mycroft) so we don't need to maintain this in two locations.

Reply:
I replaced the whole section to just point to the handbook instead :+1: